roles/hdfs-standalone/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Restart Hadoop HDFS NameNode
+  sudo: true
+  sudo_user: hdfs
+  shell: source /etc/profile.d/hadoop.sh; hadoop-daemon.sh stop namenode; hadoop-daemon.sh start namenode
```

Comment:
if one of these commands failed, the rest would continue to execute. Could you change this to use `with_items` instead? That'll give proper error messages to the user if something fails.